1 . did you run test code on you local machine? you can reproduce this error.
1. I'm not sure if this PR is the only cause, but I'm pretty sure that this PR brings at least one problem; you're using `super` (https://github.com/tork-a/rtmros_nextage/pull/239/files#diff-74831e17eddab0218a7f6d7fbc024a30R337
   ) with the object this is not inheritance of `object` class, see  HIRONX(https://github.com/start-jsk/rtmros_hironx/blob/indigo-devel/hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py#L221)    and HrpsysConfigurator (https://github.com/fkanehiro/hrpsys-base/blob/master/python/hrpsys_config.py#L161)
   which NextageClient avoid this issue with very tentative way (https://github.com/tork-a/rtmros_nextage/blob/indigo-devel/nextage_ros_bridge/src/nextage_ros_bridge/nextage_client.py#L46)

```
The model was successfully loaded ! 
Traceback (most recent call last):
  File "/home/travis/catkin_ws/install/share/nextage_ros_bridge/script/nextage.py", line 74, in <module>
    robot.init(robotname=args.robot, url=args.modelfile)
  File "/home/travis/catkin_ws/install/lib/python2.7/dist-packages/nextage_ros_bridge/nextage_client.py", line 109, in init
    HIRONX.init(self, robotname=robotname, url=url)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/hironx_ros_bridge/hironx_client.py", line 315, in init
    if set([rn[0] for rn in self.getRTCList()]).issubset(set([x.name() for x in self.ms.get_components()])) :
  File "/home/travis/catkin_ws/install/lib/python2.7/dist-packages/nextage_ros_bridge/nextage_client.py", line 344, in getRTCList
    return super(HIRONX, self).getRTCList()
TypeError: must be type, not classobj
```

--- a/nextage_ros_bridge/src/nextage_ros_bridge/nextage_client.py
+++ b/nextage_ros_bridge/src/nextage_ros_bridge/nextage_client.py
@@ -330,20 +330,18 @@ class NextageClient(HIRONX, object):
         '''
         Overwriting HrpsysConfigurator.getRTCList
         Returning predefined list of RT components.
+
+        As of March 2016, this method internally calls HIRONX.getRTCList() and
+        returns what it returns. Although we could simply remove this method
+        from NextageClient, we still keep it; it'd be easier
+        to modify an existing method than to add a new overridden method,
+        in case we might want to define RTC return list differently from HIRONX.
+
         @rtype [[str]]
         @rerutrn List of available components. Each element consists of a list
                  of abbreviated and full names of the component.
         '''
-        return [
-            ['seq', "SequencePlayer"],
-            ['sh', "StateHolder"],
-            ['fk', "ForwardKinematics"],
-            ['el', "SoftErrorLimiter"],
-            # ['co', "CollisionDetector"],
-            # ['sc', "ServoController"],
-            ['ic', "ImpedanceController"],
-            ['log', "DataLogger"]
-            ]
+        return HIRONX.getRTCList(self)
 
     def goInitial(self, tm=7, wait=True, init_pose_type=0):
         '''

